data/json/npcs/EOC_talkers/portal_storm.json

```
@@ -89,7 +89,7 @@
     "responses": [
       { "text": "No ", "topic": "TALK_PORTAL_STORM_DO_FOR_ME" },
       {
-        "text": "Will it.",
+        "text": "Will it.  (This will end your game.  Do not choose this option if you wish to continue playing.)",
```

```suggestion
        "text": "Will it.  'Your gut feels like this is jumping out of a plane without a parachute.'",
```

Something like this?